zap/src/main/java/org/zaproxy/zap/extension/ascan/ActiveScanPanel.java

```diff
@@ -250,6 +296,15 @@ private HistoryReferencesTable getMessagesTable() {
         return messagesTable;
     }
 
+    private ZapTable getFilterMessageTable() {
+        if (filterMessageTable == null) {
+            this.filterMessageTable = new ZapTable(EMPTY_FILTER_MESSAGE_MODEL);
```

Comment:
Ideally we should show the original message when the corresponding entry is selected, but something that can be added later (IMO).